### PR TITLE
Add the highlight on Secret Value & Add the documentation to using Azure Tenant URL

### PR DESCRIPTION
**Pull request by razanfawwaz**
## What kind of change does this PR introduce?

I updated the docs in the Azure Auth section,
- Added the highlight to copy the `value` from Azure, not the `Secret ID`.
- Added the documentation to using Azure Tenant URL with the image.

## What is the current behavior?

The wrong Value in Secret ID can make the error as mentioned in #1192 (Unable to exchange external code).

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: web/docs/guides/auth/auth-azure.mdx [hunk 1]
```diff
@@ -52,10 +52,19 @@ This will serve as the `client_secret` when you make API calls to authenticate t
...
+- Once the secret is generated, save the `value` (not the secret ID). 
 
 ![Obtain the client secret](/img/guides/auth-azure/azure-client-secret.png)
 
+## Obtain the Tenant URL
+
+This will using your custom Azure login page when user login.
+
+- Select the Directory (Tenant) ID value.
+- The Azure Tenat URL should look like this: `https://login.microsoftonline.com/<tenant-id>`
```
Comment (J0) [inline web/docs/guides/auth/auth-azure.mdx:64]:
```suggestion
- The Azure Tenant URL should look like this: `https://login.microsoftonline.com/<tenant-id>`
```

File: web/docs/guides/auth/auth-azure.mdx [hunk 2]
```diff
@@ -52,10 +52,19 @@ This will serve as the `client_secret` when you make API calls to authenticate t
...
 - Navigate to the "Client secrets" tab and select "New client secret".
 - Enter a description and choose your preferred expiry for the secret.
-- Once the secret is generated, save the value (not the secret ID). 
+- Once the secret is generated, save the `value` (not the secret ID). 
 
 ![Obtain the client secret](/img/guides/auth-azure/azure-client-secret.png)
 
+## Obtain the Tenant URL
+
+This will using your custom Azure login page when user login.
```
Comment (J0) [inline web/docs/guides/auth/auth-azure.mdx:61]:
```suggestion
This will allow your users to use your custom Azure login page when logging in.
```

Comment (J0) [reply web/docs/guides/auth/auth-azure.mdx:61]: Just a suggestion -- feel free to rephrase as you like

**Review by J0 — COMMENTED** [review]
Thanks for the PR! This is much needed, so appreciate that you took the time to put this together.

**Comment by razanfawwaz** [comment]
@J0 thanks for your suggestions!

**Review by J0 — APPROVED** [review]

**Review by kangmingtay — APPROVED** [review]

**Comment by razanfawwaz** [comment]
hi, @J0 @kangmingtay I made some improvements on #7905, maybe you can check the PR, thanks!

**Comment by J0** [comment]
gotcha, will look in a bit

**Pull request merged**